entrypoint.sh

```diff
@@ -76,7 +77,8 @@ function translateDockerTag() {
   elif isGitTag; then
     TAG="latest"
   elif isPullRequest; then
-    TAG="${GITHUB_SHA}"
+    BRANCH=$(echo ${GITHUB_HEAD_REF} | sed -e "s/refs\/heads\///g" | sed -e "s/\//-/g" | sed -e 's/[^ -~]//g')
```

Comment:
The logic of this line is the same as in line 69, but since the pull request context has a different reference in `GITHUB_REF`, I needed to change that to `GITHUB_HEAD_REF`

Reply:
The difference between the two vars can be read in this github docs: https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables